docs/personalization/recommendation_models.md

```diff
@@ -214,6 +214,36 @@ The logic used for resolving a submodel is as follows:
     Recommendation are fetched from all the submodels and merged based on the weight (relevance).
     If one of the submodels delivers recommendations with better relevance, the results of other models can disappear from the list.
 
+### Dynamic attributes
+
+Dynamic attribute submodels eliminate the need for manual grouping and simplify configuration.
+They allow for simpler, faster, and less demanding recommendation models building using different attributes, because all you need to do is make one request and rebuild the model.
+
+They work best in straightforward cases when you filter by the value of the attribute.
+
+Dynamic attribute submodels:
+
+- operate only on [nominal attributes](recommendation_models#nominal-attributes) (numeric attributes are not supported)
+- can be used for [popularity](recommendation_models#popularity-models) and [collaborative](recommendation_models#collaborative-models) types of models (as they support submodels)
+- have limitation of max. 50 attribute values (if more, you need to follow standard procedure - manual configuration by [[= product_name_base =]] Team)
```

Comment:
This part isn't clear to me, because in the submodel section we say:
```
Submodels must be manually configured. You do this in the property dialog of the recommendation model. After you configure the submodel, the results are generated overnight and are available on the next day.
```

So, what is the standard procedure for submodels? Manual configuration by the user or manual configuration by the Ibexa team?

Reply:
"Standard" can be tricky. I'll replace this word.